test/RBdata.sql

```diff
@@ -2400,4 +2400,18 @@ DELETE FROM `feedback_mri_comments`;
 /*!40000 ALTER TABLE `feedback_mri_comments` ENABLE KEYS */;
 UNLOCK TABLES;
+
+LOCK TABLES `visit` WRITE;
+/*!40000 ALTER TABLE `visit` DISABLE KEYS */;
+DELETE FROM `visit`;
+INSERT INTO `visit` (VisitID, VisitName) VALUES (1, 'v1'), (2, 'v2'), (3, 'v3');
```

Comment:
@PapillonMcGill All the "v1,v2,v3" should be "V1,V2,V3". 
If you want to use lower-case "v", please modify the test code.
[modules/survey_accounts/test/survey_accountsTest.php: line 214]
"Visit V1 does not exist for given candidate" to  "Visit v1 does not exist for given candidate"."

Reply:
@kongtiaowang Changed label to V1, V2 and V3 even if not related to this test for clarity

Do you know how I can obtain a mock Database object to pass to the VisitController for Travis? The test is working fine in my VM using my own version of the DB but it is not working with the mockDB for Travis (See comments under the "help I don't understand travis")?